sync/awsdatabricks.py

```diff
@@ -1,8 +1,10 @@
 import logging
 from time import sleep
+from typing import Dict, List
```

Comment:
Noticed you initially were using `dict`/`list` - just wanted to explicitly state, in case you were like "wft?", that this is for Python 3.7 support to be compatible with older Databricks versions :+1:

Reply:
Yeeeep. I remember when we upgraded from 3.7 to 3.10. I knew what was up right away.